src/tests/execute/execute_modify_tests.rs

```diff
@@ -36,6 +36,10 @@ mod execute_modify_test {
             Ok(_) => {}
         }
 
+        let exec_1_addr = deps.api.addr_make("exec_1");
+        let exec_2_addr = deps.api.addr_make("exec_2");
+        let exec_3_addr = deps.api.addr_make("exec_3");
```

Comment:
nit: might be best to just declare all these `Addr` in the scope of `mod execute_modify_test` instead of the individual test since they are constant across all the tests

Reply:
Do you mean the string value only or the `Addr`? I'm not sure of the best way to declare the addresses in that scope because they make use of the mocked `deps` in order to create the `Addr` and it doesn't seem like a we would want to define a `deps` instance at that level,

Reply:
ah ya, I had just done `Addr::unchecked();` in a package i was using and made sure the addresses matched the configured `deps.api.with_prefix($PREFIX)`, like i would use 

 `deps.api.with_prefix("tp")` for all my tests, then use valid testnet blockchain addresses for all my `Addr` variables. But it's not a big deal, i think using the `addr_make()` is best practice so this is ok